android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/customregistration/SimpleCustomRegistrationAction.kt

```diff
@@ -9,8 +9,11 @@ interface SimpleCustomRegistrationAction {
 
     fun getIdProvider(): String
 
-    fun returnSuccess(result: String?): Boolean
+    @Throws(OneginiReactNativeException::class)
```

Comment:
@Throws annotation actually has no effect in kotlin. This can be used for interoperability with java, but given that we use only kotlin, this can be removed I think